proto/metadata.proto

```diff
@@ -171,6 +171,18 @@ message Metadata {
     bool explicit_interface_in_default_vrf = 42;
     // Set to true to skip checking QOS Dropped octets stats for interface.
     bool qos_dropped_octets = 43;
+    // Device is missing subinterface packet counters for IPv4/IPv6.
+    bool subinterface_packet_counters_missing = 44;
+    // Connect-retry is not supported
+    // /bgp/neighbors/neighbor/timers/config/connect-retry.
+    bool connect_retry = 45;
+    // Set to true for device not supporting programming a gribi flow with a
```

Comment:
In a separate PR, can you remove all the "Set to true" phrases in the proto? For example, this should just be "Device does not support programming ..."

Reply:
Will work on that on here: https://github.com/openconfig/featureprofiles/pull/1764